_data/simple-icons.json

```
@@ -2634,6 +2634,11 @@
             "hex": "47848F",
             "source": "https://www.electronjs.org/"
         },
+        {
+            "title": "Electron Builder",
```

Correct me if I'm wrong but it seems their preferred formatting is `electron-builder` :thinking: 

```suggestion
            "title": "electron-builder",
```

(don't forget to update this the SVG `<title>` as well if you change this